bin/dsh

```diff
@@ -1651,7 +1653,7 @@ install_prerequisites_windows ()
 	echo-green "Installing winpty v${DSH_REQUIREMENTS_WINPTY}..."
 	(cd "$DRUDE_DOWNLOADS_DIR" && \
 		curl -sSL "$URL_WINPTY" | tar -xzf - && \
-		mv "winpty-*/bin/*" "$DRUDE_BIN_DIR/")
```

Comment:
Why is this change necessary? Or is it more like the quotes are unnecessary here?